src/store/account/actions.ts

```diff
@@ -346,6 +348,37 @@ const actions = defineActions({
 
     await api.assets.simpleTransfer(asset, to, amount);
   },
+  async getVestedTransferFee(
+    context,
+    { asset, amount, vestingPercent, unlockPeriodInDays }: VestedTransferFeeParams
+  ): Promise<Nullable<FPNumber>> {
+    try {
+      const {
+        rootState: {
+          wallet: {
+            settings: { blockNumber },
+          },
+        },
+      } = rootActionContext(context);
+      return await api.assets.getVestedTransferFee(asset, amount, blockNumber, vestingPercent, unlockPeriodInDays);
+    } catch (error) {
+      console.warn('[Vested Transfer Fee]:', error);
+      return null;
+    }
+  },
+  async vestedTransfer(
+    context,
+    { to, asset, amount, vestingPercent, unlockPeriodInDays }: VestedTransferParams
+  ): Promise<void> {
+    const {
+      rootState: {
+        wallet: {
+          settings: { blockNumber },
+        },
+      },
+    } = rootActionContext(context);
+    await api.assets.vestedTransfer(asset, to, amount, blockNumber, vestingPercent, unlockPeriodInDays);
```

Comment:
Add try catch please

Reply:
Will be handled in a method which uses it